src/backend/meet/settings.py

```diff
@@ -457,6 +457,16 @@ class Base(Configuration):
         None, environ_name="SUMMARY_SERVICE_API_TOKEN", environ_prefix=None
     )
 
+    # Marketing and communication settings
+
+    BREVO_API_KEY = values.Value(
+        None, environ_name="BREVO_API_KEY", environ_prefix=None
+    )
+    BREVO_API_CONTACT_LIST_IDS = values.ListValue(
+        [], environ_name="BREVO_API_CONTACT_LIST_IDS", environ_prefix=None
+    )
+    BREVO_API_CONTACT_ATTRIBUTES = values.DictValue({"VISIO_USER": True})
```

Comment:
Should be MARKETING_* or MARKETING_SERVICE_* or EMAILING_* instead of BREVO_*?

Reply:
The settings should match the renamed class structure and add a configuration option to specify which marketing service to use, with a factory method.